js/foam/u2/DAOListView.js

```diff
@@ -18,13 +18,26 @@
 CLASS({
   package: 'foam.u2',
   name: 'DAOListView',
-  extends: 'foam.u2.Element',
+  extends: 'foam.u2.View',
   requires: [
     'foam.u2.DetailView'
   ],
+  imports: [
+    'selection$',
+  ],
   properties: [
     {
-      name: 'data'
+      name: 'data',
```

Comment:
If you make this a DAOProperty and then listen to this.data$Proxy then it will handle all of the listener boilerplate for you.